resources/definitions/KOONOVOELF.def.json

```diff
@@ -0,0 +1,94 @@
+{
+    "version": 2,
+    "name": "KOONOVO ELF",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "KOONOVO",
+        "manufacturer": "KOONOVO",
+        "weight": 4,
+        "file_formats": "text/x-gcode",
+        "icon": "icon_ultimaker.png",
```

Comment:
This line should be removed

Reply:
> 该行应删除

delete “ icon ”： “ icon_ultimaker.png ”，